app/javascript/src/components/Dashboard/Notes/index.jsx

```diff
@@ -1,9 +1,11 @@
 import React, { useState, useEffect } from "react";
 
+// @ts-ignore
```

Comment:
Remove these. These red squiggly lines will go away in future after we add relevant changes in neetoUI.

Reply:
Will do.